_pylibczi/_pylibczi.cpp

```diff
@@ -207,6 +228,147 @@ static PyObject *cziread_allsubblocks(PyObject *self, PyObject *args) {
     return Py_BuildValue("OO", images, (PyObject *) coordinates);
 }
 
+// Begin new code
```

Comment:
As much as I appreciate comments... I don't think this comment is the most valuable 🙃

Reply:
I know it will suck, but can you basically write like a paragraph here on how reading plane coordinates works? Like a TL;DR: of CZI spec?